generators/app/templates/package.json

```diff
@@ -11,8 +11,9 @@
       <%_ if (isLibrary || isBrowser) { _%>
       "prepare": "grunt build",
       <%_ } _%>
-      "test": "check-node-version --npm 6.13.4 && TS_NODE_PROJECT='tests/tsconfig.json' TS_NODE_FILES=true nyc mocha --opts ./.mocha.opts",
-      "test:ci": "npm test -- --forbid-only"
+      "test": "check-node-version --npm 6.14.12 && TS_NODE_PROJECT='tests/tsconfig.json' TS_NODE_FILES=true nyc mocha --opts ./.mocha.opts",
+      "test:ci": "npm test -- --forbid-only",
+      "commitlint": "COMMIT_HASH=$(git log -1 --reverse --oneline --format=format:\"%h\") commitlint --from $COMMIT_HASH"
```

Comment:
It seems like this may just lint the most recent commit message instead of linting all commits from the start. If my testing is correct, `git log -1` says "get me the last commit" and `git log -1 --reverse` says "get me the last commit, then reverse that list containing one commit".

Reply:
Ha! You're right 🤦

Reply:
I updated it to use `sed`: `git log --reverse --oneline --format=format:\"%h\" | sed 1q`

This was mostly an experiment. I'm not sure we want to do this, but my feeling is the generator should create a starter project with sensible defaults, and it seemed like there's isn't any clean way to make sure linting works right away with the first commit. Any thoughts?

Reply:
What about this? Seems to work when there are multiple root commits (e.g. when repos were merged) as well as when there is only one commit in the repo.

```bash
commitlint --from $(git rev-list --max-parents=0 HEAD | sed 1q)
```

Reply:
nice! did it